core/src/components/datetime/usage/javascript.md

```diff
@@ -81,9 +92,9 @@
 ```
 
 ```javascript
-import { format, parseISO } from 'date-fns';
+import { format, parseISO } from "date-fns";
```

Comment:
Can we revert code formatting with the double/single quotes in this file (just for the JavaScript section).